pkg/azureutils/azure_disk_utils.go

```diff
@@ -596,8 +596,14 @@ func ParseDiskParameters(parameters map[string]string) (ManagedDiskParameters, e
 		case consts.ResourceGroupField:
 			diskParams.ResourceGroup = v
 		case consts.DiskIOPSReadWriteField:
+			if _, err = strconv.Atoi(v); err != nil {
+				return diskParams, fmt.Errorf("parse %s failed with error: %v", v, err)
```

Comment:
pls also provide the key name in error msg, e.g. (`parsing %s:%s failed with error: %v`, consts.DiskIOPSReadWriteField, v, err)

Reply:
add it